genestack_client/scripts/genestack_application_manager.py

```diff
@@ -386,7 +391,8 @@ def resolve_jar_file(file_path):
                 jar_files.append(os.path.join(dirpath, f))
 
     if len(jar_files) > 1:
-        raise GenestackException('More than one JAR file was found inside %s' % file_path)
+        raise GenestackException('More than one JAR file was found inside %s:\n'
+                                 '%s' % (file_path, '\n'.join(jar_files)))
```

Comment:
May be add visual indent for jars:
`'%s' % (file_path, '\n'.join(jar_files)))` - > '  %s' % (file_path, '\n  '.join(jar_files)))